app/src/main/java/org/sopt/dosopttemplate/HomeFragment.kt

```diff
@@ -42,6 +43,17 @@ class HomeFragment : Fragment() {
         val concatAdapter = ConcatAdapter(userAdapter, friendAdapter)
         //concat된 list를 fragment영역에 적용
         binding.rvFriends.adapter = concatAdapter
+
+        //fab 상단 스크롤 지원
+        upScrollListener(binding.rvFriends)
+    }
+
+    fun upScrollListener(view: RecyclerView?) {
```

Comment:
```suggestion
    fun upScrollListener() {
```
함수 내에서 `view`를 사용하지 않는 것 같은데 제거하셔도 좋을 것 같습니다.